growth/too/models.py

```diff
@@ -301,35 +301,44 @@ def gracedb(self):
     def ned_gwf(self):
         return "https://ned.ipac.caltech.edu/gwf/events"
 
+    def get_property(self, property_name, value=None):
+        root = lxml.etree.fromstring(self.content)
+        path = ".//Param[@name={}]".format(property_name)
+        elem = root.find(path)
+        value = round(float(elem.attrib.get('value', '')) * 100, 3)
+        return value
+
     @property
     def HasNS(self):
-        notice = self.gcn_notices[0]
-        root = lxml.etree.fromstring(notice.content)
-        elem = root.find(".//Param[@name='HasNS']")
-        if elem is None:
-            return None
-        else:
-            return 'HasNS: '+elem.attrib.get('value', '')
+        return self.get_property(property_name="'HasNS'")
```

Comment:
Let's move the single quotes into the format string in `get_property` so that they don't have to be repeated in each of these properties.